fullstory-segment-middleware/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
         maven { url "https://maven.fullstory.com" }
     }
 
-    compileOnly 'com.fullstory:instrumentation-full:1.4.0@aar'
+    compileOnly 'com.fullstory:instrumentation-full:1.23.0@aar'
```

Comment:
oops. this is within the middleware sub-project so that it will compile, when adding the middleware it will load from whatever FS version is actually added in the app, bump it anyways to get compile time checks - correct me if I'm wrong

Reply:
Yes, this is correct.